flink-runtime/src/main/java/org/apache/flink/runtime/entrypoint/ClusterEntrypoint.java

```diff
@@ -155,6 +156,7 @@ protected void runCluster(Configuration configuration) throws Exception {
 			// write host information into configuration
 			configuration.setString(JobManagerOptions.ADDRESS, commonRpcService.getAddress());
 			configuration.setInteger(JobManagerOptions.PORT, commonRpcService.getPort());
+			configuration.setString(RestOptions.REST_ADDRESS, commonRpcService.getAddress());
```

Comment:
This appears to be configuring the REST server's bind address, using the server address obtained from `AkkaRpcService`.   I don't understand the rationale, could you explain?

Reply:
I think what we want to get from the RestServerEndpoint is its server address. One way is to let its bind address to be the real ip of the machine. The common rpc address now is the real ip.